app/src/main/res/values/strings.xml

```diff
@@ -47,6 +47,10 @@
     <string name="username_capitalU">Username</string>
     <string name="public_habit">Public</string>
     <string name="private_habit">Private</string>
+    <string name="home_day">null</string>
+    <string name="home_welcome_message">Today will be a good day.</string>
```

Comment:
Keeping this here for future plan, will have rotating welcome message but for now this is unused.